ecs-cli/modules/cli/compose/entity/entity_test.go

```diff
@@ -58,6 +58,36 @@ func TestGetContainersForTasks(t *testing.T) {
 	assert.Equal(t, containers[0].Ec2IPAddress, aws.StringValue(ec2Instance.PublicIpAddress), "Expects PublicIpAddress to match")
 }
 
+func TestGetContainersForTasksPrivateIP(t *testing.T) {
+	containerInstanceArn := "containerInstanceArn"
+	ec2InstanceID := "ec2InstanceId"
+	ec2Instance := &ec2.Instance{PrivateIpAddress: aws.String("publicIpAddress")}
```

Comment:
This is a little confusingly named, isn't it?

Reply:
Its just the name of the function that its testing, plus "PrivateIP", which I added since the test case above it is the same except it test Public IP.

You're talking about the name of the test right? `TestGetContainersForTasksPrivateIP`?
`ContainersForTasks` is the name of the function that is being tested.

Reply:
No I was referring to the `public IpAddress` in the aws.String as a field for `PrivateIpAdress`.

Reply:
Oh oops. Lol. I'll go change it to PrivateIPAddress.